plugin_repository/api/maven_interface.md

```diff
@@ -38,7 +36,8 @@ repositories {
 }
  
 dependencies {
-  compile 'com.jetbrains.plugins:<plugin_xml_id>:<version>@<extension>'
+  compile '.jetbrains.plugins:<plugin_xml_id>:<version>' // for the plugin from Stable channel
```

Comment:
Is there a typo here for 'com.jetbrains.plugins'?

Reply:
thanks, updated